Makefile

```diff
@@ -37,11 +38,14 @@ test: unit-test e2e-test ## Run all unit tests and e2e tests
 check-licenses:
 	cd functions/ts && $(MAKE) check-licenses
 	cd functions/go && $(MAKE) check-licenses
+	cd functions/contrib/ts && $(MAKE) check-licenses
```

Comment:
@Shell32-Natsu Re. your earlier question, we only have functions written in TS in the contrib dir.